### ISSUE TITLE
Method [validateVatNumber] does not exist.

### ISSUE THREAD

**Issue by nWidart** [mpociot/vat-calculator#20]
Hello,

I just installed the package, added the service provider and alias to the app.php config file.

I would like to validate an input with the `vat_number` validation rule. For this I added `'required|vat_number'` in the rules for that field.

However with this I'm getting the following error:

```
Method [validateVatNumber] does not exist.
```

When checking the `VatCalculatorServiceProvider` class, the `registerValidatorExtension` method get's called, however the `VatCalculatorValidatorExtension@validateVatNumber` does not.

I'm using laravel `5.1.32`, with package version `1.6.3`.

Did I miss something ? 

Thank you,

Edit:

I've found that when I comment the following:

``` php
private function registerMaxFolderSizeValidator()
    {
        Validator::resolver(function ($translator, $data, $rules, $messages, $attributes) {
            return new MaxFolderSizeValidator($translator, $data, $rules, $messages, $attributes);
        });
    }
```

here: https://github.com/AsgardCms/Media/blob/master/Providers/MediaServiceProvider.php#L80

The Vat Validation rules works. Somehow laravel doesn't like multiple resolvers ?

**Comment by isbkch** [comment]
I'm having the same issue with the latest version and Laravel 5.2.23

**Comment by nWidart** [comment]
What I had to do to fix it, is to register my own custom validator with the `extend` method instead of resolve.

This is because laravel can only have one resolver, meaning if this is used in multiple places, they will get overwritten by each other.
It would probably be best if this package also uses `extend`

**Comment by isbkch** [comment]
Hum, I see. Would you be able to do a pull a request ?

**Comment by mpociot** [comment]
The main reason I chose the `resolve` approach was because I wanted to provide custom error messages.

How would this work with `extend`?

**Comment by nWidart** [comment]
@mpociot I see. I'm currently looking at [image-validator](https://github.com/cviebrock/image-validator/blob/master/src/ImageValidatorServiceProvider.php) package, which uses custom translations, with the extend method.

So I assume it is possible indeed.

@isbkch you can see on [this commit](https://github.com/AsgardCms/Media/commit/5a443348eb23d629e57eca3fd02de93c3870fb3a) what I had to change in order to use extend.

**Comment by mpociot** [comment]
Fixed with version 1.7.1

**Comment by nWidart** [comment]
Awesome, thank you! 😄
